libfwupdplugin/fu-device-locker.c

```diff
@@ -180,9 +180,13 @@ fu_device_locker_new_full(gpointer device,
 	if (!self->open_func(device, error)) {
 		g_autoptr(GError) error_local = NULL;
 		if (!self->close_func(device, &error_local)) {
-			if (!g_error_matches(error_local, FWUPD_ERROR, FWUPD_ERROR_NOTHING_TO_DO)) {
-				g_debug("ignoring close error on aborted open: %s",
-					error_local->message);
+			if (error_local != NULL) {
```

Comment:
If `self->close_func` returns with FALSE then it has to set a GError -- it's how GLib works -- it's not optional.